<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="fileRefresh">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>새로고침</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileDelete">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제하기</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fileStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>고유번호</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>파일명</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>업로드</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MD5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>파일 크기</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="reportList">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reportDelete">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportRefresh">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>새로고침</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="reportContent">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="reportStatus">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>270</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reportBan">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>차단하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportSave">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>112</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>저장하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileReport">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>신고하기</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
